SPECS/xcp-ng-pv-tools.spec

```diff
@@ -272,6 +272,9 @@ install -D -m755 %{SOURCE3} %{buildroot}/opt/xensource/libexec/unmount_xstools.s
 /opt/xensource/libexec/unmount_xstools.sh
 
 %changelog
+* Fri Nov 14 2025 Gael Duperrey <gduperrey@vates.tech> - 8.3-14
```

Comment:
When I add a temporary changelog entry that is going to get modified in the next commit, I like to add ` - WIP` at the end (and then I remove it in the next commit). It's not a formal rule, though. The idea is that if we look at a specific commit which adds a changelog entry, we can know whether it's the definitive changelog entry or not.

Reply:
And that should be better. Is it ok for you now?